tools/psa/build_tfm.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Can we not be compatible with both python2 and python3?

Reply:
@Patater With python2 going out of support in Jan2020, I'd prefer not to keep on supporting python2. This is one way of saying script is only compatible with python3.

Reply:
This may prevent us from merging into Mbed OS `master`, which still supports `python2`. I'm not sure when `python2` will be dropped from Mbed OS `master`.